src/layouts/job.astro

```diff
@@ -0,0 +1,42 @@
+---
+import MainLayout from '@/layouts/main.astro';
+import { Navbar } from '@/components/ui/navbar';
+import Footer from '@/components/ui/footer.astro';
+
+const { title } = Astro.props;
+---
+
+<MainLayout title={title}>
+  <Navbar client:load />
+  <article
+    class="prose prose-invert mx-auto w-11/12 py-9 lg:pt-20 xl:w-10/12 2xl:w-[1280px]"
+  >
+    <h1 class="m-0! text-center">{title}</h1>
+    <div class="py-8 text-center">
```

Comment:
why do we have duplicate text-center classes? Also I don't think we should center everything (only the title should be centered imo).

Reply:
Fixed.